docs/source/index.rst

```diff
@@ -38,11 +30,16 @@ Guide
    api
    license
 
+
 Contact
 -------
 For suggestions, queries or appreciation feel free to contact `Dr.JasonKwan <https://apps.pharmacy.wisc.edu/sopdir/jason_kwan/>`_
 at jason.kwan@wisc.edu
 
+Copyright and License
+---------------------
+Autometa is copyright 2020 Ian J. Miller, Evan R. Rees, Izaak Miller and Jason C. Kwan, 
+and is released under the GNU Affero General Public License v3 (see :ref:`License`).
```

Comment:
So copyright is also in the footer of each page. I'm not sure it is necessary at this point to be hardcoded in here. Note, Ian and Izaak will need to be added to the other copyright section.